<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWindow</class>
 <widget class="QDialog" name="SetupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setup</string>
  </property>
  <widget class="QTabWidget" name="preferencesTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="graphViewPreferenceTab">
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <attribute name="title">
     <string>Graph View Preferences</string>
    </attribute>
    <widget class="QPushButton" name="defaultGraphViewButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Default Settings</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="graphArrowMovementScaleGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="title">
      <string>Graph Arrow Movement Scale</string>
     </property>
     <widget class="QSlider" name="arrowMSHorizontalSlider">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>451</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
      <property name="sliderPosition">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="arrowMSSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="graphViewGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>521</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="title">
      <string>Graph View Parameters</string>
     </property>
     <widget class="QGroupBox" name="xAxisGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>501</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="title">
       <string>X-Axis View</string>
      </property>
      <widget class="QLabel" name="keyViewScalingLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>View Scaling: </string>
       </property>
      </widget>
      <widget class="QSlider" name="keyScalingSlider">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>421</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="keyScalingSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="suffix">
        <string>0%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="yAxisGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>501</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="title">
       <string>Y-Axis View</string>
      </property>
      <widget class="QSlider" name="valueMaxSlider">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="valueMaxSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="suffix">
        <string> (g's)</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
       <property name="value">
        <number>7</number>
       </property>
      </widget>
      <widget class="QLabel" name="valueMaxLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Maximum Value:</string>
       </property>
      </widget>
      <widget class="QLabel" name="valueMinLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Minimum Value:</string>
       </property>
      </widget>
      <widget class="QSlider" name="valueMinSlider">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>401</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="minimum">
        <number>-7</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>-7</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="valueMinSpinBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="suffix">
        <string> (g's)</string>
       </property>
       <property name="minimum">
        <number>-7</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="value">
        <number>-7</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="okGrapheViewButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>380</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelGraphViewButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>380</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="labelPreferencesTab">
    <attribute name="title">
     <string>Labeling Preferences</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
